packages/ios-client/Sources/FishjamClient/webrtc/PeerConnectionManager.swift

```diff
@@ -176,11 +176,9 @@ internal class PeerConnectionManager: NSObject, RTCPeerConnectionDelegate {
         config.candidateNetworkPolicy = .all
         config.tcpCandidatePolicy = .disabled
 
-        // if ice servers are not empty that probably means we are using turn servers
+        // if ice servers are not empty that probably means we are using turn servers, which are not handled at the moment
         if iceServers.count > 0 {
```

Comment:
**suggestion:** 
If they are empty, then we can just pass this empty list from protobuf message
```suggestion
```